contribution_guide.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to uTensor
+
+Welcome to uTensor contribution guide. Want to make things better? [Talk to us](utensor@googlegroups.com)!
+
+## Ways to Contribute
+
+### Two kinds of pull request
+
+  1. Adding new features to uTensor:
+    -  Create a new [issue](https://github.com/uTensor/uTensor/issues) and discuss the idea with core maintainers. We want to make sure your time is well spent and learn about you.
```

Comment:
Changing the last sentence:
"We want to make sure your time is well spent and learn about you."
to
"Through the discussions, we want to make sure your time is well spent and learn about you."